packages/amplify-e2e-core/src/init/deleteProject.ts

```diff
@@ -10,16 +10,19 @@ import { getBackendAmplifyMeta } from '../utils';
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const deleteProject = async (cwd: string, profileConfig?: any, usingLatestCodebase = false): Promise<void> => {
   // Read the meta from backend otherwise it could fail on non-pushed, just initialized projects
-  const { StackName: stackName, Region: region } = getBackendAmplifyMeta(cwd).providers.awscloudformation;
-  await retry(
-    () => describeCloudFormationStack(stackName, region, profileConfig),
-    stack => stack.StackStatus.endsWith('_COMPLETE') || stack.StackStatus.endsWith('_FAILED'),
-  );
-
-  const noOutputTimeout = 1000 * 60 * 20; // 20 minutes;
-  return spawn(getCLIPath(usingLatestCodebase), ['delete'], { cwd, stripColors: true, noOutputTimeout })
-    .wait('Are you sure you want to continue?')
-    .sendConfirmYes()
-    .wait('Project deleted locally.')
-    .runAsync();
+  try {
```

Comment:
we must exit gracefully even if delete fails; otherwise artifacts won't get stored properly by our test runner